fem/gridfunc.hpp

```diff
@@ -131,11 +131,16 @@ class GridFunction : public Vector
    /// Read only access to the (optional) internal true-dof Vector.
    /** Note that the returned Vector may be empty, if not previously allocated
        or set. */
-   const Vector &GetTrueVector() const { return t_vec; }
```

Comment:
The new behavior doesn't align with the doxygen, same for the non-const version.